README.md

```diff
@@ -91,6 +91,33 @@ For each test, you should see output like
     Latency     9.58ms    6.03ms  94.90ms   85.57%
     Req/Sec   9936.90   2222.61  10593.00     95.24
 ```
+
+#### Measuring energy consumption 
+
+You can also measure energy consumption, rather than throughput. 
+This tooling is still evolving,  so the current process is a bit manual and needs several windows. The current tooling requires MacOS or Linux.
+
+1. Clone the `power-server` utilities, https://github.com/metacosm/power-server. These provide an hardware-independent abstraction layer over hardware power metric tooling. 
```

Comment:
I'm experimenting with JBang configuration for the power consumption tool:
`jbang https://github.com/metacosm/power-server/blob/polish/cli/src/main/java/net/laprun/sustainability/cli/Power.java -c "<command to run>"` should do the trick for now, at least, until I validate the approach and create a JBang catalog for easier use.